python/samples/concepts/auto_function_calling/chat_gpt_api_function_calling.py

```diff
@@ -64,7 +64,7 @@
     temperature=0.7,
     top_p=0.8,
     function_call_behavior=FunctionCallBehavior.EnableFunctions(
-        auto_invoke=True, filters={"included_plugins": ["math", "time"]}
+        auto_invoke=False, filters={"included_plugins": ["math", "time"]}
```

Comment:
Is this change on purpose? Asking because doesn't seem related to this PR, I thought it might be a local debugging change committed by mistake.

Reply:
Good point, will revert